crates/red_knot_python_semantic/src/types.rs

```diff
@@ -2204,6 +2210,7 @@ impl<'db> KnownInstanceType<'db> {
             Self::NoReturn => KnownClass::SpecialForm,
             Self::Never => KnownClass::SpecialForm,
             Self::Any => KnownClass::Object,
+            Self::Tuple => KnownClass::Object,
             Self::Type => KnownClass::Object,
```

Comment:
Oh I think I missed this in your earlier PR adding support for `typing.Type`. Shouldn't both of these branches be `KnownClass::SpecialForm`? The symbols are defined as instances of `_SpecialForms` in typeshed: https://github.com/python/typeshed/blob/2666d3b5b06b5436470017d46d01929da70ab5f8/stdlib/typing.pyi#L207-L212

```suggestion
            Self::Tuple => KnownClass::SpecialForm,
            Self::Type => KnownClass::SpecialForm,
```

Reply:
In fact, I think this might be what's now causing the new false positive error in the `tomllib` benchmark

Reply:
Yeah, for better or worse, typeshed doesn't reflect all the gory details of the runtime implementation right now. In some cases, like this, it simplifies a little. It might seem strange, but it's more important here that we follow what typeshed says the facts are rather than follow the actually true facts as they are at runtime.